### ISSUE TITLE
Issue with az Devops Seuciryt Permission Update on the "Project" Namespace

### ISSUE THREAD

**Issue by silverscout** [Azure/azure-cli#23759]
Can you please provide an example on how to update the Project Namespace.  I run the following and it says, or atleast, I believe it is completing fine.  However, when I go to the permissions page, nothing is actually set.  

az devops security permission update `
--id $namespaceId #namespaceID of the Project
--subject $subject  #descriptor of the group starting in vssgp
--Token $tkn
--Allow-bit $bit # is the BIT number I am adding allow to 
--org https://dev.azure.com/$org
--merge true

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1121ea02-7c35-1af4-0226-e277c8d1d566
* Version Independent ID: 5697a5d6-2a89-282c-1b83-22bf82068a6c
* Content: [az devops security permission](https://docs.microsoft.com/en-us/cli/azure/devops/security/permission?view=azure-cli-latest)
* Content Source: [latest/docs-ref-autogen/devops/security/permission.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/latest/docs-ref-autogen/devops/security/permission.yml)
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by SaurabhSharma-MSFT** [comment]
@silverscout We are looking into it and get back to you for any additional information.

**Comment by yonzhan** [comment]
route to CXP team

**Comment by silverscout** [comment]
@SaurabhSharma-MSFT  and @yonzhan 
I found the answer in another issue.  
https://github.com/Azure/azure-cli/issues/23537

btw.. should find a way to make tokens easier.   using a different token for different namespaces is painful to script for.  haha.

**Comment by SaurabhSharma-MSFT** [comment]
@silverscout Good to know that you are unblocked. We will now proceed to close this thread. If there are further questions regarding this matter, please reopen it and we will gladly continue the discussion.
